Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -31,6 +31,9 @@ class CommandLineConfigLayerLoader final : public Config::ConfigLayerLoader
     if (!audio_backend.empty())
       m_values.emplace_back(Config::MAIN_DSP_HLE.location, ValueToString(audio_backend == "HLE"));
 
+    if (batch)
```

Comment:
Do you think we should leave a comment on _why_ batch mode disables render-to-main?